composer.json

```
@@ -39,11 +39,11 @@
         "ext-json": "*",
         "enlightn/security-checker": "^1.1|^2.0",
         "guzzlehttp/guzzle": "^7.0",
-        "larastan/larastan": "^2.0",
+        "larastan/larastan": "^3.0",
```

It should be this, no?

```suggestion
        "larastan/larastan": "^2.0|^3.0",
```